server_tests/test/clientTests.js

```diff
@@ -1337,7 +1337,7 @@ describe("RUST API TEST SUITE", function() {
             return Utility.restoreDefaultOptions(defaultOptionsFile)
             .then(function() {
                 return Utility.deleteAllImages();
-            });
+            }, wrapError);
```

Comment:
Some of the others have nested promises eg. 
```
afterEach( function() {
            return Utility.checkActiveSession()
            .then( function(isActive) {
                if (isActive) {
                    return testClient.closeSession(sessionId)
                    .then( function onSuccess (res) {
                        validate.done(res.body, schema.names.commandCloseSession);
                    }, wrapError);
                }
            }, wrapError);
        });
```

Should they be
```
afterEach( function() {
            return Utility.checkActiveSession()
            .then( function(isActive) {
                if (isActive) {
                    return testClient.closeSession(sessionId)
                    .then( function onSuccess (res) {
                        validate.done(res.body, schema.names.commandCloseSession);
                    });
                }
            })
            .catch(wrapError)
        });
```
for consistency?